### DISCUSSION TITLE
About the maintenance of this project

### DISCUSSION THREAD

**Discussion by croisez** [mcguirepr89/BirdNET-Pi#1124] [General]
Hello birdnet-pi enthusiasts,
I was wondering if there are still people maintaining the source code tree of this nice project.
I tell you that because this morning I went in the Pull requests tab, and saw PR not handled for a long time : mine has been posted there on October, and there is one even dated from May.
What is the point ? Who is maintaining the code ? I know that Patrick is not on the wave anymore, but there were other people, is'nt it ?

**Comment by Svardsten53** [comment]
Please see #1003

**Reply by croisez** [reply]
Ok, the final word is: there is no more maintainers.
So what ?
Is it possible that someone else be promoted as maintainer ?
